checker/tests/nullness/Issue2432b.java

```diff
@@ -0,0 +1,42 @@
+// Additional test cases for issue 2432 (unrelated to poly annotations)
```

Comment:
Isse #2432 is about poly annotations, so the `unrelated to poly annotations` is confusing.
Can you make clearer what these tests are about? Is it uses of Poly in arrays?